### PR TITLE
changing to state update semantics instead of end state

### PR DESCRIPTION
**Pull request by balopat**

### CODE REVIEW

File: pkg/minikube/tunnel/tunnel.go [hunk 1]
```diff
@@ -18,6 +18,12 @@ type LoadBalancerTunnelState struct {
 	ServicesWithLoadBalancer []core_v1.Service
 }
 
-func (tunnelState *LoadBalancerTunnelState) next() *LoadBalancerTunnelState {
-	return tunnelState
+type LoadBalancerTunnelStateUpdate struct {
```
Comment (aaron-prindle) [inline pkg/minikube/tunnel/tunnel.go:21]: nit:  this name is a bit verbose, perhaps TunnelStateUpdate, LBTunnelStateUpdate, or LoadBalancerUpdate would make sense for these?

**Review by aaron-prindle — APPROVED** [review]
LGTM

File: pkg/minikube/tunnel/tunnel.go (hunk 1 above)
Comment (balopat) [reply pkg/minikube/tunnel/tunnel.go:21]: sounds good, we can rename it!

Comment (balopat) [reply pkg/minikube/tunnel/tunnel.go:21]: `TunnelStateUpdate`

**Pull request merged**